fix: remote_utility typo in remote_access_software.csv

sventec opened the pull request:
### Details

PR fixes a typo in the `remote_access_software.csv` lookup file. Executable name is currently `NinjaRMMAgenPatcher.exe`, PR corrects typo `Agen` --> `Agent`.

Additional fix added for `.ese` --> `.exe` for `nxservice*.ese`.

### Checklist

- [ ] Validate name matches `<platform>_<mitre att&ck technique>_<short description>` nomenclature
- [ ] [CI/CD](https://github.com/splunk/security_content/actions) jobs passed ✔️ 
- [ ] Validated SPL logic.
- [ ] Validated tags, description, and how to implement.
- [ ] Verified references match analytic.
- [ ] Confirm updates to lookups are handled properly.

### Notes For Submitters and Reviewers

- If you're submitting a PR from a fork, ensuring the box to allow updates from maintainers is checked will help speed up the process of getting it merged.
- Checking the output of the `build` CI job when it fails will likely show an error about what is failing. You may have a very descriptive error of the specific field(s) in the specific file(s) that is causing an issue. In some cases, its also possible there is an issue with the YAML. Many of these can be caught with the pre-commit hooks if you set them up. These errors will be less descriptive as to what exactly is wrong, but will give you a column and row position in a specific file where the YAML processing breaks. If you're having trouble with this, feel free to add a comment to your PR tagging one of the maintainers and we'll be happy to help troubleshoot it.
- Updates to existing lookup files can be tricky, because of how Splunk handles application updates and the differences between existing lookup files being updated vs new lookups. You can read more [here](https://docs.splunk.com/Documentation/SplunkCloud/8.2.2203/Admin/PrivateApps#Manage_lookups_in_Splunk_Cloud_Platform) but the short version is that any changes to lookup files need to bump the datestamp in the lookup CSV filename, and the reference to it in the YAML needs to be updated.

sventec committed:
fix: remote_utility typo in remote_access_software.csv

sventec commented:
May require updates in the lookup's related YAML file, or other changes. I avoided making further changes there as I'm unsure on what exactly is required. Happy to adjust as needed, or just file this as an issue if that would be preferred.

ljstella commented:
Hey @sventec, thanks for the change! 

Regarding your comment, if you could bump the version and date in `remote_access_software.yml` to go along with your change, that will ensure we generate a lookup with a new filename (needed for in-place upgrades of ESCU).

sventec commented on lookups/remote_access_software.csv:
```diff
@@ -61,7 +61,7 @@ Netviewer (GoToMeet),,nvClient.exe,Algorius*,netviewer,TRUE,RMM,Obsolute - found
-NinjaRMM,*.ninjarmm.com,NinjaRMMAgenPatcher.exe,Ninja MSP*,,TRUE,RMM,https://www.ninjaone.com/faq/,2/9/2024
+NinjaRMM,*.ninjarmm.com,NinjaRMMAgentPatcher.exe,Ninja MSP*,,TRUE,RMM,https://www.ninjaone.com/faq/,4/4/2025
 NoMachine,user_managed,nomachine*.exe,NoMachine*,nomachine,TRUE,RMM,https://kb.nomachine.com/AR04S01122,2/9/2024
 NoMachine,,nxservice*.ese,,,TRUE,RMM,https://kb.nomachine.com/AR04S01122,2/9/2024
```
Just noticed this typo as well: `nxservice*.ese` (`ese` not `exe`).. Separate PR? Issue? Not worth updating? Unless this is actually referencing Extensible Storage Engine file(s)..

ljstella replied:
Feel free to include the changes here in this PR.

sventec replied:
Will do, thanks!

sventec committed:
fix: addtl remote_utility typo in remote_access_software.csv

sventec committed:
chore: update lookup date and version

sventec commented:
> Hey @sventec, thanks for the change!
> 
> Regarding your comment, if you could bump the version and date in `remote_access_software.yml` to go along with your change, that will ensure we generate a lookup with a new filename (needed for in-place upgrades of ESCU).

Thanks @ljstella, added that additional typo correction, and updated date & version in the associated YAML file. Appreciate the assistance wrt. the lookup update process!

ljstella reviewed (approved):
LGTM

patel-bhavin committed:
Merge branch 'develop' into patch-1
